src/commands/Minion/use.ts

```diff
@@ -171,6 +172,30 @@ export default class extends BotCommand {
 			.split(',')
 			.map(part => part.trim().toLowerCase())
 			.map(getOSItem);
+
+		// Redying
+		if (items.length === 2) {
+			// e.g. [Blood Dye, Dwarven warhammer (blood)]
+			const baseItem = dyedItems.find(i => i.dyedVersions.some(o => items.includes(o.item)));
+			const dyeToApply = baseItem?.dyedVersions.find(i => items.includes(i.dye));
+			const dyedVariantTheyHave = baseItem?.dyedVersions.find(i => items.includes(i.item));
+			if (baseItem && dyeToApply && dyedVariantTheyHave) {
+				await msg.confirm(
+					`Are you sure you want to use a ${dyeToApply.dye.name} on your ${dyedVariantTheyHave.item.name}?`
+				);
+				if (!msg.author.owns(dyedVariantTheyHave.item.id)) {
```

Comment:
Need to check user's bank for both the target item AND the dye itself.

Attempting to do: `-use blood dye, dwarven warnana` without a blood dye yields an unhelpful error, `An unexpected error occurred :RSSad:` and an exception is thrown.